libr/include/r_util/r_str.h

```diff
@@ -26,6 +26,7 @@ static inline void r_str_rmch(char *s, char ch) {
 	}
 }
 
+#define R_STR_CPY(dest, src) (R_SHOULD_BE_ARRAY (dest), r_str_ncpy (dest, src, sizeof (dest)))
```

Comment:
if the dest is not an array i assume it will fail at compile time, so the macro name is confusing, because it doesn't implies that. so it's imho an overengineering solution for a problem that doesn't exist and it is just used in 2 places.